specs/StoragePickers/PickerLocationId.md

```diff
@@ -0,0 +1,110 @@
+
+# Fields
+
+| **Name**           | **Value** | **Description**                                                 |
+|--------------------|-----------|-----------------------------------------------------------------|
+| DocumentsLibrary   | 0         | [The Documents library](#pickerlocationiddocumentslibrary)      |
+| ComputerFolder     | 1         | [This PC](#pickerlocationidcomputerfolder)                      |
+| Desktop            | 2         | [The Desktop](#pickerlocationiddesktop)                         |
+| Downloads          | 3         | [The Downloads folder](#pickerlocationiddownloads)              |
+| HomeGroup          | 4         | [The HomeGroup folder, (avoid using this)](#pickerlocationidhomegroup) |
+| MusicLibrary       | 5         | [The Music library](#pickerlocationidmusiclibrary)              |
+| PicturesLibrary    | 6         | [The Pictures library](#pickerlocationidpictureslibrary)        |
+| VideosLibrary      | 7         | [The Videos library](#pickerlocationidvideoslibrary).           |
+| Objects3D          | 8         | [The 3D Objects folder, (avoid using this)](#pickerlocationidobjects3d)|
+| Unspecified        | 9         | [An unspecified location](#pickerlocationidunspecified)         |
+
+# Definition
+
+```C#
+enum PickerLocationId
+{
+    DocumentsLibrary = 0,
+    ComputerFolder = 1,
+    Desktop = 2,
+    Downloads = 3,
+    HomeGroup = 4,
+    MusicLibrary = 5,
+    PicturesLibrary = 6,
+    VideosLibrary = 7,
+    Objects3D = 8,
+    Unspecified = 9,
+}
+```
+
+## PickerLocationId.DocumentsLibrary
+The `DocumentsLibrary` value represents the user's Documents library.
+
+This is the default location where many applications save documents, and it is often mapped 
+to the `%USERPROFILE%\Documents` folder on the user's system. 
+If the user has configured OneDrive for documents, this location would be redirected to a OneDrive 
+folder, such as `%USERPROFILE%\OneDrive - ...\Documents`.
+
+## PickerLocationId.ComputerFolder
+The `ComputerFolder` value represents "This PC" directory.
+
+This location provides access to all drives and devices connected to the user's computer, 
+including internal and external storage devices.
+
+## PickerLocationId.Desktop
+The `Desktop` value represents the user's Desktop.
+
+This is the location where users often save shortcuts, files, and folders for quick access. 
+It is often mapped to `%USERPROFILE%\Desktop`.
+
+## PickerLocationId.Downloads
+The `Downloads` value represents the user's Downloads folder.
+
+This is the default location where web browsers and other applications save downloaded files. 
+It is often mapped to `%USERPROFILE%\Downloads`.
+
+## PickerLocationId.HomeGroup
+! Note that [HomeGroup has been removed from Windows 10 (version 1803) and later](https://support.microsoft.com/en-us/windows/homegroup-removed-from-windows-10-version-1803-07ca5db1-7bca-4d11-68a3-a31ff4a09979), 
```

Comment:
What does it map to? If this isn't meaningful, then it is probably better to just remove this enum value.